### DISCUSSION TITLE
use library as a DALI monitor

### DISCUSSION THREAD

**Discussion by dan-s-github** [sde1000/python-dali#129] [General]
Hi there, thanks for making this library publicly available

It is used by the  hasseb.fi USB DALI Master Controller Software which I used to test drivers and lights before they were installed in the house, which has a KNX setup with 2 KNX-DALI Gateways.
 
As I have some troubles using miboxer DT-8 controllers with the current gateway I need to troubleshoot some of the telegrams sent from the gateway to controller and also check the responses sent back to the gateway.

Would it be possible to use this library as a monitoring/logging device only using the hasseb or any other low cost interface?

click boards mentioned in other discussion would seem a viable option if the can be used as Dali logger.

**Comment by sde1000** [comment]
I don't believe the Hasseb device will let you do this — it's physically capable of it, like all DALI interfaces, but the firmware doesn't support it.

Currently I think the `async-buswatch.py` script has only been tested with the Tridonic/Lunatone DALI-USB interface. It works well, but isn't particularly low-cost.

It should be possible to make Lunatone LUBA devices work, but I can't help with that because I don't have any.

**Reply by dan-s-github** [reply]
As you mentioned in other discussion it might need more than just a [Mikroe USB Adapter](https://www.mikroe.com/click-usb-adapter) and a [DALI 2 Click](https://www.mikroe.com/dali-2-click) to get the DALI telegrams.

If I get a Lunatone DALI-USB interface then I would probably just use the Lunatone DALI Cockpit to validate the telegram flow.
